bin/rollem.js

```diff
@@ -6,7 +6,7 @@ const fs = require('fs')
 const join = require('path').join
 const argv = require('minimist')(process.argv.slice(2))
 
-const configFilename = argv.c || 'rollem.config.js'
+const configFilename = (argv.c === true) ? 'rollem.config.js' : argv.c;
```

Comment:
@YuryStrozhevsky We have tools that can take care of that for us. We just have to understand how to use a knife properly. :-) In addition our future selves will love us more if we ever decide to lint our code.

My go to is I just follow @bahmutov's conventions to the best of my abilities while always improving the code is my go to mission. Your mileage may vary.

https://medium.com/@goatslacker/no-you-dont-need-semicolons-148d936b9cf2#.4kk91y888
https://www.codecademy.com/en/forum_questions/507f6dd09266b70200000d7e